webapp/app/templates/content/contact.html

```diff
@@ -6,19 +6,19 @@ <h1 class="my-4">{{ _('contact.heading') }}</h1>
     </div>
 
     <div class="mb-5">
-        <h3>{{ _('contact.service-headline') }}</h3>
+        <h2>{{ _('contact.service-headline') }}</h2>
         <p class="mt-2">{{ _('contact.service-infopage', link=url_for('howitworks')) }}</p>
         <p>{{ _('contact.service-mail') }}</p>
         <p>{{ _('contact.service-legal') }}</p>
     </div>
 
     <div class="mt-5 mb-5">
-        <h3>{{ _('contact.feedback-headline') }}</h3>
+        <h2>{{ _('contact.feedback-headline') }}</h2>
         <p class="mt-2">{{ _('contact.feedback-text') }}</p>
     </div>
 
     <div class="mt-5 mb-5">
-        <h3>{{ _('contact.security-headline') }}</h3>
+        <h2>{{ _('contact.security-headline') }}</h2>
```

Comment:
This does also change the look. So we might have to either also use the h3 class here or think of another approach?